client/src/locales/languages.js

```diff
@@ -24,7 +24,7 @@ export const localization = {
   ru: "ru",
   "te-IN": "te-IN",
   th: "th",
-  tr: "tr",
+  "tr-TR": "tr-TR",
```

Comment:
Wouldn't this need to be changed in line 51 as well. probably in gtm constants and the other languages file where the JSONs are imported.

Reply:
yes, I think It should be changed for all other languages too. But I couldn't test them all.  I changed this because of local language of the browser. 
Just "tr" would not work for below line 10,

![code](https://user-images.githubusercontent.com/34690158/104924978-3ea3c080-59af-11eb-9712-48270c2851c5.png)

I haven't checked line 51 and I don't know where it is used. I'll check that.

Reply:
Are we changing Turkish to tr-tr? If so, we will definitely need to change in GTM constants and also the json file (on Lokalise). Please let me know if we are changing it!

Reply:
It is such a mistake. I don't understand. Now it's working or maybe there is an edge case and I couldn't find it again. Anyway, I discarded this change.